format_nomad_with_env.sh

```diff
@@ -68,19 +69,38 @@ fi
 # We need to specify the database and Nomad hosts for development, but
 # not for production because we just point directly at the RDS/Nomad
 # instances.
+# Conversely, in prod we need AWS credentials and a logging config but
+# not in development.
```

Comment:
We definitely still need to be able to log and have AWS credentials in development!

If you don't want to have a awslogger, we should check for something like `local` rather than `prod`.